tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -118,7 +96,7 @@ db, err := sql.Open("mysql", "<usename>:<your_password>@tcp(<host>:4000)/test?tl
 
 <div label="Node.js">
 
-[Mysql2](https://www.npmjs.com/package/mysql2)'s TLS connection configurations are used here as an example.
+[Mysql](https://www.npmjs.com/package/mysql2)'s TLS connection configurations are used here as an example.
```

Comment:
why this? Its name is mysql2.

Reply:
This link is original link，I am not sure that [Mysql2](https://www.npmjs.com/package/mysql2) is correct link